### DISCUSSION TITLE
Parse data after edit/paste

### DISCUSSION THREAD

**Discussion by benediktvaldez** [iddan/react-spreadsheet#401] [Ideas]
Is there a way to provide a custom function to parse **pasted** data? Similar to [`parsePaste` in react-datasheet](https://github.com/nadbm/react-datasheet#options)

Is there a way to provide a custom function to parse **edited** data? As in, when the user is done editing and exits edit mode, I'd like to validate the value and possibly autoformat it for example.

I'd actually love to be able to set a “valid/invalid” state per cell (I'm doing it now via custom `DataViewer` which is fine I guess)

**Comment by iddan** [comment]
Great idea, I'm pretty busy right now to implement it myself but PRs are welcome
